src/language-js/printer-estree.js

```diff
@@ -5633,11 +5633,13 @@ function isTemplateLiteral(node) {
   return node.type === "TemplateLiteral";
 }
 
-function isIdentiferAsync(node) {
+// `inject` is used in AngularJS 1.x, `async` in Angular 2+
+// example: https://docs.angularjs.org/guide/unit-testing#using-beforeall-
+function isIdentiferInjectOrAsync(node) {
```

Comment:
What do you think about remaining this to something that specifies its purpose instead of just what it tests for?

Reply:
fixed, pls have a look